kdc/kdc.8

```diff
@@ -155,6 +155,25 @@ Permit tickets with no addresses.
 This option is only relevant when check-ticket-addresses is TRUE.
 .It Li allow-anonymous = Va boolean
 Permit anonymous tickets with no addresses.
+.It Li historical_anon_realm = Va boolean
+Enables pre-7.0 non-RFC-comformant KDC behavior.
+With this option set to
+.Li true
+the client realm in anonymous pkinit as-replies will be the requested realm,
+rather than the RFC-conformant
+.Li WELLKNOWN:ANONYMOUS
+realm.
+This can have a security impact on servers that expect to grant access to
+anonymous, but authenticated to the KDC users of the realm in question.
```

Comment:
s/anonymous, but authenticated/anonymous-but-authenticated/

Reply:
Also, replace the sentence-ending period with a colon.

Reply:
Sure